### DISCUSSION TITLE
Simply delete data

### DISCUSSION THREAD

**Discussion by lunchboxer** [beginner-corp/begin-community#102] [Ideas] [answered]
I can see in the dashboard a table of the data for my app. Is there a simple way to truncate the table, to just nuke the data?

**Comment by tbeseda** [comment, marked as answer]
Hey @lunchboxer 
I realize our reply is really late here, sorry for the delay.

At this point there is no way to update or delete data in your app from the Begin.com dashboard. Begin is able to _read_ the data, but cannot write or destroy data.
I could see this being a feature added in the future. So, for now I can move this to the Ideas category so we can consider it as we build out the Data feature.
